content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -197,15 +197,15 @@ for i in range(1, EPOCHS+1):
     metrics = evaluate(model, x_test, y_test)
     for k, v in metrics.items():
         print('Epoch %s: %s=%s'%(i, k, v))
-+       dvclive.log(k, v)
++       dvclive.log_metric(k, v)
```

Comment:
Not blocking: should we go ahead and update `dvclive` to `live` here since that's the naming used everywhere else?